README.md

```
@@ -1,29 +1,25 @@
+2. Compiler: This component translates the AST produced by Magik-Tools' Magik-Squid into JVM bytecode using the Class-File API.
 
    Currently, only supports literal types `string`, `integer` and `float`; as well as the `invoke` expressions.
 3. Runtime: provides a standard library with basic functionality to run a Magik program.
 
    As of now, only the `write` variable in the `sw` package has a barebones implementation.
 
-
-
 Features
+
 -------
```

Markdownlint (0.37.0)

Expected: ----------; Actual: -------
Horizontal rule style

(MD035, hr-style)